apps/editors/templates/editors/review.html

```diff
@@ -163,25 +163,20 @@ <h3 id="history">
       </div>
 
       <div class="review-actions-section review-actions-files data-toggle"
-           data-value="{{ actions_minimal|join("|") }}"{% if allow_unchecking_files %} data-uncheckable="1"{% endif %}>
-        <label for="id_addon_files"><strong>{{ form.addon_files.label }}</strong></label>
+           data-value="{{ actions_minimal|join("|") }}">
+        <label for="id_addon_files"><strong>{{ _('Files:') }}</strong></label>
         <ul>
-            {% for pk, label in form.fields.get('addon_files').choices %}
+            {% for file in form.helper.version.unreviewed_files %}
             <li>
-              <label for="file-{{ pk }}"{% if pk in form.addon_files_disabled %} class="light"{% endif %}>
-                <input id="file-{{ pk }}" type="checkbox" value="{{ pk }}" name="addon_files"
-                       {% if pk in form.addon_files_disabled %}disabled=""{% endif %} />
-                {{ label }}
+                <label for="file-{{ file.pk }}">
+                <input type="checkbox" checked disabled />{# only there for visual retro-compat ;) #}
```

Comment:
It was only for "visual retro compatibility" as mentioned in the comment, but that might be stupid. I can remove that.